archlinuxcn/llvm-git/PKGBUILD

```diff
@@ -121,7 +121,7 @@ package_lldb-git() {
 
     pkgdesc="Next generation, high-performance debugger (git version)"
     url="https://lldb.llvm.org/"
-    depends=("clang-git=$pkgver-$pkgrel" "llvm-libs-git=$pkgver-$pkgrel")
+    depends=("clang-git=$pkgver-$pkgrel" "llvm-libs-git=$pkgver-$pkgrel" "python" "libedit" "libxml2")
```

Comment:
The official `lldb` do not have `libedit` and `libxml2` as dependency and they are already dependency of `llvm-libs-git`. Are these just to make sure all directly linked packages are listed as dependencies? (It's not arch's official policy though I'm good with that.)